components/features/VideoConference/Controls/Controls.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import styles from "./Controls.module.css";
+import { motion } from "framer-motion";
+import { useAVToggle } from "@100mslive/react-sdk";
+import VideoConferenceIcon from "@components/shared/svg/AppIcons/VideoConferenceIcon";
+import MicrophoneIcon from "./MicrophoneIcon";
+import LeaveIcon from "./LeaveIcon";
+
+interface Props {
+  onLeave?: () => void;
+}
+
+const Controls: React.FC<Props> = ({ onLeave }) => {
+  const { isLocalVideoEnabled, isLocalAudioEnabled, toggleAudio, toggleVideo } =
+    useAVToggle((e) => { console.log(e.name) });
```

Comment:
Hello, rashad. I think there's a console.log here, you might have forgot to remove. Maybe next commit you can remove it. 😀

Reply:
Hello Alex, that console log is actually intentional to notify the client of any errors for the time being. In the next iteration I will be implementing a toast to notify of any errors :D

Reply:
Oh okay rashad, so for now the errors are not displayed to the users directly but only through the console. Thanks for letting me know. 😁